deltachat-ffi/deltachat.h

```diff
@@ -5106,6 +5159,15 @@ void dc_event_unref(dc_event_t* event);
  */
 #define DC_EVENT_SECUREJOIN_JOINER_PROGRESS       2061
 
+
+/**
+ * The connectivity to the server changed.
+ * This means that you should refresh the connectivity view
+ * and possibly the connectivtiy HTML; see dc_get_connectivity() and
+ * dc_get_connectivity_html() for details.
+ */
+#define DC_EVENT_CONNECTIVITY_CHANGED             2062
```

Comment:
maybe use sth. as 2100 and leave a little gap here as we've done for the other events, so if we add more SECUREJOIN events, they can have a 2062ff.